backoff.opam

```diff
@@ -1,15 +1,15 @@
 # This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
-synopsis:
-  "Exponential backoff mechanism for ocaml"
+synopsis: "Exponential backoff mechanism for ocaml"
 maintainer: ["Carine Morel <carine@tarides.com>"]
 authors: ["KC Sivaramakrishnan <sk826@cl.cam.ac.uk>"]
 license: "ISC"
-homepage: "https://github.com/lyrm/backoff"
-bug-reports: "https://github.com/lyrm/backoff/issues"
+homepage: "https://github.com/ocaml-multicore/backoff"
+bug-reports: "https://github.com/ocaml-multicore/backoff/issues"
 depends: [
   "dune" {>= "3.3"}
-  "ocaml" {>= "5.0"}
+  "ocaml" {>= "4.13"}
```

Comment:
Was this the lowest 4.x version that works? I think this could probably build on lower versions.

Reply:
Mmmh, I did not really think this though about it, I just ... copied/pasted from `kcas` probably ! I will make some checks.

Reply:
`domains-shims` works for OCaml from `4.12`, so I guess this may be the lower limit.

Reply:
> val min : int -> int -> int
> 
> Return the smaller of the two arguments.
> 
>     Since 4.13.0

Actually `4.13` is right.

Reply:
(Also I guess, we could use `Stdlib.min` for older OCaml)